### PR TITLE
MODCLUSTER-806 Change the mod_proxy_cluster license to ASL 2.0

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/MODCLUSTER-806

Goals:

* Update the main repo licence at root - that gets picked up by GitHub UI
* Update headers on source files + build scripts
* Keep tests as they are now

### CODE REVIEW

**Comment by rhusar** [comment]
[ERROR] Failed to execute goal on project mod_cluster_test: Could not resolve dependencies for project org.jboss.mod_cluster:mod_cluster_test:pom:2.0.4.Final-SNAPSHOT: The following artifacts could not be resolved: org.jboss.mod_cluster:mod_cluster-container-spi:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-core:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-container-tomcat-8.5:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-load-spi:jar:2.0.4.Final-SNAPSHOT: Could not find artifact org.jboss.mod_cluster:mod_cluster-container-spi:jar:2.0.4.Final-SNAPSHOT -> [Help 1]

Our scripts rely on a version set to be the same as in main mod_cluster branch... we should rely on released versions unless necessary otherwise.

**Comment by rhusar** [comment]
@jajik Since you already have experience with reviewing these - please have a look to make sure I didn't miss something. Thanks! 

BTW This time I found it more readable to do an empty space after the licence header.

**Comment by jajik** [comment]
> [ERROR] Failed to execute goal on project mod_cluster_test: Could not resolve dependencies for project org.jboss.mod_cluster:mod_cluster_test:pom:2.0.4.Final-SNAPSHOT: The following artifacts could not be resolved: org.jboss.mod_cluster:mod_cluster-container-spi:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-core:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-container-tomcat-8.5:jar:2.0.4.Final-SNAPSHOT, org.jboss.mod_cluster:mod_cluster-load-spi:jar:2.0.4.Final-SNAPSHOT: Could not find artifact org.jboss.mod_cluster:mod_cluster-container-spi:jar:2.0.4.Final-SNAPSHOT -> [Help 1]
> 
> Our scripts rely on a version set to be the same as in main mod_cluster branch... we should eventualy rely on released versions unless necessary otherwise.

A PR is already opened https://github.com/modcluster/mod_proxy_cluster/pull/174

I agree that we should rely on the released ones in the future. I plan to do it with some other changes regarding a better handling of the tests dependencies.

File: native/scripts/windows-build.bat [hunk 1]
```diff
@@ -1,8 +1,11 @@
+REM Copyright The mod_cluster Project Authors
+REM SPDX-License-Identifier: Apache-2.0
+
 REM kudos to Michal Karm Babacek <karm@fedoraproject.org>
 REM this script is from (https://github.com/modcluster/ci.modcluster.io)
 
 REM You need to define HTTPD_DEV_HOME environment variable to point to
-REM your httpd's folder if you run this scirpt on you own.
+REM your httpd's folder if you run this script on you own.
```
Comment (jajik) [inline native/scripts/windows-build.bat:8]: Good catch.

**Review by jajik — APPROVED** [review]
LGTM

File: native/scripts/windows-build.bat (hunk 1 above)
Comment (rhusar) [reply native/scripts/windows-build.bat:8]: :-)

**Pull request merged**